### DISCUSSION TITLE
Using Mocaccino to create my own distro

### DISCUSSION THREAD

**Discussion by mappo-mocaccino** [mocaccinoOS/mocaccino#117] [Q&A]
Hi, I am new to mocaccino but not new to linux. I have previous experience using Gentoo for over 10 years but haven't used it (much) for about 10 years. I am very intrigued by this new distro and wish to determine if it is a good choice for me to create my own distro, which is my new project. It seems like a good way to go from what I have read. 

I left Gentoo for one main reason, which is power consumption. I discovered that compiling packages on each system was inefficient and consumed a lot of electricity. While I like the flexibility of the customization offered by portage the power bill was huge when using it on multiple systems. This disadvantage has effects on both my power bill and on the environment of our planet.

I am now in the process of planning a distro targeted at a particular user demographic and wish to use a system like (what I believe) mocaccino offers to compile packages to create a customized distro. Is this a good base for this type of a project or am I better off simply customizing another existing distro such as debian?

Any feedback is appreciated.

**Comment by vargux** [comment]
Hi,
You can take a preview with "Calculate Linux"  it is based on the Gentoo Linux.
https://www.calculate-linux.org/

Greetings!
